packages/client/src/components/CategoriesListDropDown/CategoriesListDropDown.component.js

```diff
@@ -42,8 +39,8 @@ const DropDownView = ({
 DropDownView.propTypes = {
   options: PropTypes.arrayOf(PropTypes.string).isRequired,
   lable: PropTypes.string.isRequired,
-  onSelect: PropTypes.string.isRequired,
-  showFilterIcon: PropTypes.string.isRequired,
+  onSelect: PropTypes.func.isRequired,
+  showFilterIcon: PropTypes.bool.isRequired,
```

Comment:
Not sure why you set it as `isRequired` when you also have a default value above? 

Decide, either it's optional and the users of this component don't have to care, or it's required and they have to pass it in (fix it where it's missing instead).

Reply:
Its show error: propType "onSelect" is not required, but has no corresponding defaultProps declaration. same error in propType "showFilterIcon" is not required, but has no corresponding defaultProps declaration.  I am struggling to pass defaultProps declaration.

Reply:
https://github.com/HackYourFuture-CPH/fp-class22/blob/develop/packages/client/src/components/Button/Button.component.js#L49-L54